beacon_node/eth1/tests/test.rs

```diff
@@ -53,7 +54,7 @@ fn random_deposit_data() -> DepositData {
 /// Blocking operation to get the deposit logs from the `deposit_contract`.
 async fn blocking_deposit_logs(eth1: &GanacheEth1Instance, range: Range<u64>) -> Vec<Log> {
     get_deposit_logs_in_range(
-        &eth1.endpoint(),
+        &SensitiveUrl::parse(eth1.endpoint().as_str()).unwrap(),
```

Comment:
This is just a matter of style (and I don't suggest changing it), but often you can convert `String` -> `&str` by writing `&`, like `&eth1.endpoint()`.